src/background/browserAction.ts

```diff
@@ -38,23 +38,23 @@ chrome.browserAction.onClicked.addListener(tab => {
                 chrome.notifications.create({
                     iconUrl: chrome.extension.getURL("logo.png"),
                     type: "basic",
-                    title: "WatchParty",
+                    title: "StreamParty",
```

Comment:
thx for these updates. Since this `title:` property is rendered, can we use a space? I think it's the only user-facing spot that doesn't.